OvieMudi/Banka
# #165330785 create new bank account endpoint

#### What does this PR do?
- Adds a new endpoint for creating user accounts
- Adds an endpoint for getting all accounts in DB
- Add new test cases for accounts route
#### Description of Task to be completed?
- Make sure the following endpoints are working:
```
POST /api/v1/accounts
GET /api/v1/accounts
```
#### How should this be manually tested?
- Start the local server and test the endpoint via Postman
- Run  ` npm test ` and ensure all tests are passing
#### Any background context you want to provide?
A user(client) will be able to create a new bank account from this endpoint
#### What are the relevant pivotal tracker stories?
[#165330785 ](https://www.pivotaltracker.com/story/show/165330785 )
